pkg/util/interrupt/interrupt.go

```diff
@@ -65,7 +65,10 @@ func (h *Handler) Signal(s os.Signal) {
 func (h *Handler) Run(fn func() error) error {
```

Comment:
@smarterclayton would also be nice to have package and method level documentation comments that were missing when this package was introduced in #21624, WDYT?